### ISSUE TITLE
Each time I launch Renovate I have 2 PR created

### ISSUE THREAD

**Issue by jycouet** [renovateapp/renovate#1653]
#### This is a:

* [ ] Bug report (non-security related)
* [ ] Feature request
* [x] I'm not sure which of those it is

#### I'm using:

* [ ] The Renovate GitHub App
* [ ] Self-hosted GitHub
* [ ] Self-hosted GitLab
* [x] Self-hosted VSTS

#### Please describe the issue:

Hello, I'm not sure if it's a problem of configuration or VSTS or ??? but each time I launch the Renovate Process it's creating 2 new PR (with valid dependencies to update). I launch it again just after, and again 2 new valid PR... Why the first Renovate didn't do directly the 4 PR?

I'm missing something, but I don't know what.

Could you help me please?

Thank you

**Comment by rarkins** [comment]
Hi, I think we need the debug log - can you email it to me if you want to save time redacting it?

Also could be helpful if you can work out exactly or approximately which release this started happening on.

I'm curious also how it's possible. Do those PRs have unique branch names? On GitHub and GitLab we can't have redundant branch names so you wouldn't normally find identical PRs.

**Comment by jycouet** [comment]
Hello, 
Sorry for the delay... I had a small look, and I think I found something.

In the log, here is what I can find:
```
2018-03-21T02:01:52.5075150Z  INFO: Processing 53 branch(es) (repository=test/WebApp)
2018-03-21T02:01:52.5088277Z  INFO: PR hourly limit remaining: 2 (repository=test/WebApp)
2018-03-21T02:01:52.5101113Z DEBUG: Enforcing prConcurrentLimit (20) (repository=test/WebApp)
```

Here is my renovate.json
```
{
  "extends": [
    "config:base"
  ]
}
```

And I can see in the logs the preset
`2018-03-21T02:01:31.7126318Z DEBUG: Found preset :prHourlyLimit2 (repository=test/WebApp)`

Then it's simple, how should I override `prHourlyLimit` in `renovate.json`?

Thank you

**Comment by rarkins** [comment]
Yes, if you want unlimited PRs per run then set `"prHourlyLimit": 0`

**Comment by rarkins** [comment]
I thought you'd meant duplicate PRs created every time :)

**Comment by jycouet** [comment]
thx @rarkins
